pombola/south_africa/views/api.py

```diff
@@ -0,0 +1,21 @@
+from django.http import JsonResponse
+
+from pombola.core.models import Organisation, ContactKind
+
+
+# Output Popolo JSON suitable for WriteInPublic for any committees that have an
+# email address.
+def committees_popolo_json(request):
```

Comment:
I'd slightly prefer this to be a class-based view than a function, for consistency with the rest of `pombola.south_africa.views` - there are function-based views in Pombola core, but generally, I think we're trying to move towards class-based views. (It doesn't really matter for a view as simple as this, but I think they help to make the code clearer in more complex cases.) There's a similar example to this in `id-mapping-store`, which might be a useful comparison: https://github.com/mysociety/id-mapping-store/blob/627a9b07f06122da7a80cfbf3bb75fab36b90ef4/id_mappings/views.py#L118-L134